core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -592,7 +593,12 @@ public SystemSessionProperties(
                 dataSizeProperty(
                         QUERY_MAX_TOTAL_MEMORY_PER_NODE,
                         "Maximum amount of total memory a query can use per node",
-                        nodeMemoryConfig.getMaxQueryTotalMemoryPerNode(),
```

Comment:
This seems like a bug. Undo? We already have `QUERY_MAX_MEMORY_PER_NODE` above that uses `getMaxQueryMemoryPerNode`